inst/extdata/sectoral/mappingCEDS2024toREMIND.csv

```diff
@@ -17,13 +17,13 @@ CEDS2024;REMIND;IAMC;CommentREMIND;CommentIAMC
 1A2g_Ind-Comb-transpequip;Energy|Demand|Industry|Other;Industrial Sector;;
 1A2g_Ind-Comb-wood-products;Energy|Demand|Industry|Other;Industrial Sector;;
 1A3ai_International-aviation;Transport|Pass|Aviation|International|Demand;Aircraft;Transport emissions variables from EDGE-T reporting do not comply with emissions reporting structure so far, once REMIND names adapted -> adapt here as well;
-1A3aii_Domestic-aviation;Energy|Demand|Transport;Aircraft;;
-1A3b_Road;Energy|Demand|Transport;Transportation Sector;;
-1A3c_Rail;Energy|Demand|Transport;Transportation Sector;;
+1A3aii_Domestic-aviation;w/o Bunkers|Energy|Demand|Transport;Aircraft;;
+1A3b_Road;w/o Bunkers|Energy|Demand|Transport;Transportation Sector;;
```

Comment:
is this really the way the variable should be called?

Reply:
The mapping leaves out the ``Emi|<pollutant>`` prefixes, so the final names after the calc function will be the same as on REMIND side after this PR: https://github.com/pik-piam/remind2/pull/709